fe/fe-core/src/test/java/com/starrocks/sql/plan/ReplayFromDumpTest.java

```diff
@@ -126,6 +131,18 @@ private String getDumpInfoFromFile(String fileName) throws Exception {
                         getExplainString(TExplainLevel.COSTS));
     }
 
+    private Pair<QueryDumpInfo, String> getPlanFragment(String dumpJsonStr, SessionVariable sessionVariable,
```

Comment:
It looks like getCostPlanFragment，you can modify getCostPlanFragment directly

Reply:
Yeah, but you know, choose the plan by mock cost is really troublesome